VultisigApp/VultisigApp/Views/Vault/TokenSelectionView.swift

```diff
@@ -28,31 +32,33 @@ struct TokenSelectionView: View {
                 NavigationBackSheetButton(showSheet: $showTokenSelectionSheet)
             }
         }
+        .task {
+            try? await tokenViewModel.loadData(chain: group.chain)
```

Comment:
don't use `try?` , please capture the error appropriately , it swallow the error